core/trino-main/src/main/java/io/trino/operator/DirectExchangeClientConfig.java

```diff
@@ -140,4 +141,16 @@ public DirectExchangeClientConfig setAcknowledgePages(boolean acknowledgePages)
         this.acknowledgePages = acknowledgePages;
         return this;
     }
+
+    public DataSize getDeduplicationBufferSize()
+    {
+        return deduplicationBufferSize;
+    }
+
+    @Config("exchange.deduplication-buffer-size")
+    public DirectExchangeClientConfig setDeduplicationBufferSize(DataSize deduplicationBufferSize)
+    {
+        this.deduplicationBufferSize = deduplicationBufferSize;
```

Comment:
do we need `requireNotNull` here? I am not sure. It is not common, but I don't know if this is an omission or not.

Reply:
For stats we usually prefer validation by `@NotNull` that I forgot to add :-)